src/main/java/org/eclipse/tractusx/wallet/stub/credential/CredentialService.java

```diff
@@ -184,12 +186,26 @@ private CustomCredential issueBpnCredential(String holderBpn, DidDocument issuer
         Map<String, Object> subject = new HashMap<>();
         subject.put(StringPool.ID, holderDocument.getId());
         subject.put(StringPool.HOLDER_IDENTIFIER, holderBpn);
-        subject.put(StringPool.TYPE, StringPool.BPN_CREDENTIAL);
         subject.put(StringPool.BPN, holderBpn);
         CustomCredential credentialWithoutProof = CommonUtils.createCredential(issuerDocument.getId(),
                 vcIdUri.toString(), StringPool.BPN_CREDENTIAL, DateUtils.addYears(new Date(), 1), subject);
 
         memoryStorage.saveCredentials(vcId, credentialWithoutProof, holderBpn, StringPool.BPN_CREDENTIAL);
         return credentialWithoutProof;
     }
+
+    private CustomCredential issueDataExchangeGovernanceCredential(String holderBpn, DidDocument issuerDocument, DidDocument holderDocument, URI vcIdUri, String vcId) {
+        Map<String, Object> subject = new HashMap<>();
+        subject.put(StringPool.ID, holderDocument.getId());
+        subject.put(StringPool.HOLDER_IDENTIFIER, holderBpn);
+        subject.put(StringPool.GROUP, "UseCaseFramework");
```

Comment:
Since this value will not be used elsewhere, it is considered unnecessary to define it as a constant. However, if you recommend defining it as a constant, I will make the change.